### PR TITLE
PP rule support

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/31923

## Description
Added support to pp rule as content item to the upload to MP flow.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

### CODE REVIEW

**Comment by Noy-Maimon** [comment]
Successful build with RN aggregation of two PP rules (under the pack `Zoom`): https://app.circleci.com/pipelines/github/demisto/content/109213/workflows/808263c7-beb0-4484-92dd-131fcf88d674/jobs/415377

File: Tests/Marketplace/marketplace_services.py [hunk 1]
```diff
@@ -1636,6 +1637,10 @@ def collect_content_items(self):
                             'name': content_item.get('name', ""),
                             'description': content_item.get('description', ""),
                         })
+                    elif current_directory == PackFolders.PREPROCESS_RULES.value:
+                        folder_collected_items.append({
+                            'name': content_item.get('scriptName', "") #TODO: verify, couldnt find in scheme
+                        })
```
Comment (guyfreund) [inline Tests/Marketplace/marketplace_services.py:1643]:
```suggestion
                            'name': content_item.get('scriptName', "")
                        })
```
what about description?

Comment (Noy-Maimon) [reply Tests/Marketplace/marketplace_services.py:1643]: PP rule was not added to the scheme yet, will wait with this PR until the scheme is clear.

Comment (Noy-Maimon) [reply Tests/Marketplace/marketplace_services.py]: Verified by the scheme in the pp rule server PR

**Review by guyfreund — APPROVED** [review]
🆒

**Pull request merged**